meta.yaml

```
@@ -0,0 +1,37 @@
+package:
+  name: "pytest-playwright"
+  version: "{{ environ.get('GIT_DESCRIBE_TAG') }}"
```

```suggestion
  version: "{{ environ.get('GIT_DESCRIBE_TAG') | replace('v', '') }}"
```

This should work I believe. Since it uses jinja2.

---

Cheers! I didn't realise it was Jinja, I should've recognised the double curlies.